src/commands/init.js

```diff
@@ -0,0 +1,258 @@
+import asyncCommand from '../lib/async-command';
+import fs from 'fs.promised';
+import copy from 'recursive-copy';
+import glob from 'glob';
+import mkdirp from 'mkdirp';
+import ora from 'ora';
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import promisify from 'es6-promisify';
+import path from 'path';
+import { install, initialize, pkgScripts, initGit, trimLeft } from './../lib/setup';
+
+const TEMPLATES = {
+	full: 'examples/full',
+	empty: 'examples/empty',
+	root: 'examples/root',
+	simple: 'examples/simple'
+};
+
+export default asyncCommand({
+	command: 'init',
+
+	desc: 'Create a new application interactively',
+
+	builder: {
+		default: {
+			description: 'Use default values',
+			type: 'boolean',
+			alias: 'y',
+			default: false
+		}
+	},
+
+	async handler(argv) {
+		const questions = [
+			{
+				type: 'input',
+				name: 'name',
+				message: 'Directory and package name for the app',
+				default: 'my_app'
```

Comment:
when we use `preact create` command for creating a project, it has `name` and `dest` key. Since those were flags, `dest` key was set to `<name>` itself.

In other words, whatever you set for `name` key, that will be set for `dest` too.